db/migrate/20150508124600_copy_unmanaged_hosts_to_interfaces.rb

```diff
@@ -19,9 +19,13 @@ def up
     say "Migrating Unmanaged Host interfaces to standalone Interfaces"
 
     FakeHost.where(:managed => false).each do |host|
+      say "  Migrating #{host.name}"
       nic = FakeNic.where(:host_id => host.id, :primary => true, :provision => true).first
       if nic.present?
         nic.name = host.name if nic.name.blank?
+        # we ignore validation errors on existing primary interface
+        say "Host #{host.name} has invalid interface #{nic.identifier}, you have to fix it manually through API" unless nic.valid?
```

Comment:
I'd extend a bit more and explain how to possibly fix it. Maybe writing a short troubleshooting page in the wiki and linking to it would work. I'm just afraid people will see this and freak out not knowing exactly what's invalid in that interface and how to fix it.